src/lib/services/graphql/graphql.service.ts

```diff
@@ -315,11 +320,16 @@ export class GraphqlService {
         code: event.code,
         reason: event.reason,
       });
-      if (event.code === CLIENT_SIDE_CLOSE_EVENT) {
-        this.setConnectionStatus(ConnectionStatus.DISCONNECTED);
-        this.clearMonitoring();
-        this.socket = null;
-        return;
+
+      this.setConnectionStatus(ConnectionStatus.DISCONNECTED);
+      this.socket = null;
+
+      this.clearMonitoring();
```

Comment:
⚠️ When a socket closes, the ping interval should be cancelled as well. Because it will retry connection anyway following the logic below.

HOWEVER, monitoring for window `"offline"` event will also be disabled, which is undesirable, because after the user returns online we should try to reconnect as soon as possible.